recipes-devtools/fastfetch/files/LICENSE

```diff
@@ -0,0 +1,22 @@
+MIT License
```

Comment:
I see that the fastfetch repo has a LICENSE file already. You don't need to include a copy of it here in the layer. Just need to make sure the hash in `LIC_FILES_CHKSUM` matches the hash for the LICENSE file in the source repo.

Unless this was included here for another reason? On quick glance it looks identical to the one from the source repo.